migtests/lib/common.py

```diff
@@ -0,0 +1,15 @@
+import os
+import sys
+import argparse
+
+test_type_flags = {}
+
+def valparser():
```

Comment:
nit: you can return the test_type_flags variable instead of keeping it global in this module. 
Generally, global variables are avoided because they can be confusing or error-prone.